make/configure.cmake.h

```diff
@@ -110,6 +110,8 @@
 #cmakedefine ANNOTATIONS
 #cmakedefine HAVE_RSEQ
 #cmakedefine HAVE_HALF_FLOAT
+#cmakedefine BUILD_PT_POST_PROCESSOR
```

Comment:
These are added but not used anywhere: I assume added as a mistake and should be removed.

Reply:
These are intended to be used to conditionally compile code that invokes drpttracer and drpt2ir during online trace and offline post-processing. I suppose they can be added in the next PR when we actually add that new code.

Reply:
I removed them. And I will add them in next PR.